hles/transformation/src/test/scala/org/broadinstitute/monster/dap/EnvironmentTransformationsSpec.scala

```diff
@@ -13,13 +13,18 @@ class EnvironmentTransformationsSpec extends AnyFlatSpec with Matchers with Opti
     val mapped = EnvironmentTransformations.mapEnvironment(
       RawRecord(
         1,
-        Map("redcap_event_name" -> Array("baseline_arm_1"), "baseline_complete" -> Array("2"))
+        Map(
```

Comment:
Just flagging that you will probably see a merge conflict here with some changes I pushed recently (removing baseline_arm_1 logic from env transformations)